content/copilot/using-github-copilot/ai-models/changing-the-ai-model-for-copilot-chat.md

```diff
@@ -120,9 +120,9 @@ For more information about these models, see:
 
 ## Changing the AI model for {% data variables.product.prodname_copilot_chat_short %}
 
-These instructions are for {% data variables.product.prodname_vs %}. For {% data variables.product.prodname_vscode_shortname %} or for {% data variables.product.prodname_copilot_short %} on the {% data variables.product.github %} website, click the appropriate tab at the top of this page.
+These instructions are for the {% data variables.product.prodname_vs %}. For instructions on different clients, click the appropriate tab at the top of this page.
 
-To use multi-model {% data variables.product.prodname_copilot_chat_short %}, you must use {% data variables.product.prodname_vs %} 2022 version 17.12 or later. See the [{% data variables.product.prodname_vs %} downloads page](https://visualstudio.microsoft.com/downloads/).
+> [!IMPORTANT] To use multi-model {% data variables.product.prodname_copilot_chat_short %}, you must use {% data variables.product.prodname_vs %} 2022 version 17.12 or later. See the [{% data variables.product.prodname_vs %} downloads page](https://visualstudio.microsoft.com/downloads/).
```

Comment:
Let me know if I shouldn’t add it, but it seems crucial because the article all about it